cirq/circuits/qasm_output.py

```diff
@@ -308,7 +308,11 @@ def on_stuck(bad_op):
                 main_op, keep=keep, fallback_decomposer=fallback, on_stuck_raise=on_stuck
             )
 
-            should_annotate = decomposed != [main_op]
+            qasms = list(
```

Comment:
idiomatic python rarely uses `list(map(lambda`. Consider writing as a list comprehension

Reply:
Fixed in latest commit